tests/common/beamlines/test_device_instantiation.py

```diff
@@ -6,6 +6,8 @@
 from dodal.common.beamlines import beamline_utils
 from dodal.utils import BLUESKY_PROTOCOLS, make_all_devices
 
+ALL_BEAMLINES = {"adsim", "i03", "i04", "i04_1", "i23", "i24", "p38", "p45"}
```

Comment:
Looks like these tests now automatically populate the list of all beamlines, so this isn't required

Reply:
I thought it will be added every time the tests runs and git will show it as file changed ..so was thinking it will be better if it is in github..Can remove it if required

Reply:
It's been replaced with a call to this function

https://github.com/DiamondLightSource/dodal/blob/32ff2397a4e6c9d4f420c02182ffd0bbd02f8ad4/src/dodal/beamlines/__init__.py#L18